manifests/s/SvenRowoldt/MidiKey2Key/2.3.5.0/SvenRowoldt.MidiKey2Key.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: SvenRowoldt.MidiKey2Key
+PackageVersion: 2.3.5.0
+PackageLocale: en-US
+Publisher: Sven Rowoldt

```

```suggestion
Publisher: zero Sound & Equipment, Sven Rowoldt
```
This should match control panel